media/js/mozorg/internet-health/privacy-day-egg.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, KonamiCode) {
+    'use strict';
+
+    var $document = $(document);
+    var $body = $('body');
+    var $kzp = $('#kzp');
+
+    new KonamiCode(function() {
+        $kzp.fadeIn('fast', function() {
```

Comment:
By no means a blocker, but it would be nice to avoid jQuery animations and just go full CSS?